modules/cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk_{context}"]
+= Disabling CPU load balancing for DPDK
```

Comment:
Yeah, this has been here for ages.. but as you asked below, this feature is mostly used with busy loop workloads. Because it only disables cpu load balancing = the scheduler tick.